AppInspector.CLI/CLICmdOptions.cs

```diff
@@ -31,15 +62,9 @@ public class CLIAnalyzeCmdOptions : CLICommandOptions
         [Option('r', "custom-rules-path", Required = false, HelpText = "Custom rules file or directory path")]
         public string? CustomRulesPath { get; set; }
 
-        [Option('h', "match-depth", Required = false, HelpText = "First match or best match based on confidence level (first|best)", Default = "best")]
-        public string MatchDepth { get; set; } = "best";
-
         [Option('i', "ignore-default-rules", Required = false, HelpText = "Exclude default rules bundled with application", Default = false)]
         public bool IgnoreDefaultRules { get; set; }
 
-        [Option('d', "allow-dup-tags", Required = false, HelpText = "Output contains unique and non-unique tag matches", Default = false)]
-        public bool AllowDupTags { get; set; }
-
         [Option('b', "suppress-browser-open", Required = false, HelpText = "Suppress automatically opening HTML output using default browser", Default = false)]
```

Comment:
How do you feel about removing this entirely? It seems kinda spooky to have a CLI open a web browser by default. Users can just view the HTML file if they want.

Reply:
I'm fine with removing it.